gprofiler/merge.py

```diff
@@ -29,16 +28,9 @@
 
 logger = get_logger_adapter(__name__)
 
-SAMPLE_REGEX = re.compile(
-    r"\s*(?P<comm>.+?)\s+(?P<pid>[\d-]+)/(?P<tid>[\d-]+)(?:\s+\[(?P<cpu>\d+)])?\s+(?P<time>\d+\.\d+):\s+"
-    r"(?:(?P<freq>\d+)\s+)?(?P<event_family>[\w\-_/]+):(?:(?P<event>[\w-]+):)?(?P<suffix>[^\n]*)(?:\n(?P<stack>.*))?",
-    re.MULTILINE | re.DOTALL,
-)
-
 # ffffffff81082227 mmput+0x57 ([kernel.kallsyms])
 # 0 [unknown] ([unknown])
 # 7fe48f00faff __poll+0x4f (/lib/x86_64-linux-gnu/libc-2.31.so)
-FRAME_REGEX = re.compile(r"^\s*[0-9a-f]+ (.*?) \(\[?(.*?)\]?\)$")
 
 
 def parse_one_collapsed(collapsed: str, add_comm: Optional[str] = None) -> StackToSampleCount:
```

Comment:
Not sure what to do with these three functions. I don't think they belong here. Maybe somewhere in utils?

Reply:
I mean: `parse_one_collapsed`, `parse_one_collapsed_file `and `parse_many_collapsed`

Reply:
Yeah, utils. You can add a file `gprofiler/utils/collapsed_format.py` and put them there